keyless/pepper/example-client-rust/src/main.rs

```diff
@@ -164,31 +155,31 @@ async fn main() {
         .await
         .unwrap();
     assert_eq!(StatusCode::OK, raw_response.status());
-    let pepper_response = raw_response.json::<PepperResponseV1>().await.unwrap();
```

Comment:
I guess the struct renames are not gonna break any compatibility since everything is JSON-serialized and struct names are not included?

Reply:
Correct afaict